parsl/dataflow/strategy.py

```diff
@@ -154,13 +151,13 @@ def unset_logging(self):
 
         self.logger_flag = True
 
-    def _strategy_simple(self, status_list, tasks, *args, kind=None, **kwargs):
-        self._general_strategy(status_list, tasks, *args, strategy_type='simple', kind=None, **kwargs)
+    def _strategy_simple(self, status_list, tasks):
+        self._general_strategy(status_list, tasks, strategy_type='simple')
 
-    def _strategy_htex_auto_scale(self, status_list, tasks, *args, kind=None, **kwargs):
-        self._general_strategy(status_list, tasks, *args, strategy_type='htex', kind=None, **kwargs)
+    def _strategy_htex_auto_scale(self, status_list, tasks):
+        self._general_strategy(status_list, tasks, strategy_type='htex')
 
-    def _general_strategy(self, status_list, tasks, *args, strategy_type='simple', kind=None, **kwargs):
+    def _general_strategy(self, status_list, tasks, *, strategy_type):
```

Comment:
`*` also needs to be removed, I assume.

Reply:
The `*` is to keep strategy_type be a kwarg, like it was before this PR.